qemu/tests/cfg/hotplug_mem_stress.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_mem_stress:
+    type = hotplug_mem_stress
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
+    no Host_RHEL.6
+    no RHEL.5 RHEL.6 RHEL.7
+    no Windows
+    # Notes:
+    #    The threshold on ppc was confirmed with developer
+    #    since crashkernel size is much bigger than x86
+    ppc64,ppc64le:
+        threshold = 0.15
+    guest_numa_nodes = "node0"
+    mem_devs = "mem0"
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    numa_memdev_node0 = mem-mem0
+    backend_mem = memory-backend-ram
+    target_mems = "plug1"
+    size_mem_plug1 = 1G
+    stress_dir = "/tmp/stress-ng"
+    get_stress-ng = "[ -d ${stress_dir} ] && rm -rf ${stress_dir}; git clone --depth=1 git://kernel.ubuntu.com/cking/stress-ng.git ${stress_dir}"
```

Comment:
Good ~

Reply:
It is better to use `get_stress_ng`

Reply:
Could you pls tell why we need convert '-' to '_' ?

Reply:
> Could you pls tell why we need convert '-' to '_' ?

The recommend naming rule is:  start with a letter, underline (_), and can be followed by any number of letters, numbers, and underscores (_)